build/android/build.sh

```diff
@@ -47,16 +44,14 @@ elif [[ "$LC_UNAME" == "darwin" ]]; then
 fi
 source `dirname $0`/../common/build-common.sh
 
-# Only update and install the NDK if necessary, as this can be slow
+# Unless explicitly specified, NDK version will be set to match exactly the required one
+export FILAMENT_NDK_VERSION=${FILAMENT_NDK_VERSION:-$(cat `dirname $0`/ndk.version)}
```

Comment:
Does this need to be exported here? The root build.sh already exports it

Reply:
Why not? root `build.sh` accepts it from the environment.